app/views/provider/admin/backend_apis/metrics/new.html.erb

```diff
@@ -0,0 +1,11 @@
+<% content_for :sublayout_title, @metric.child? ? "New Method" : "New Metric" %>
+
+<%= semantic_form_for @metric,
+  :url => if @metric.child?
```

Comment:
Could we define the url in a different line? It would look a bit cleaner in my opinion

Reply:
Done